aep/general/0132/aep.md.j2

```diff
@@ -126,8 +130,8 @@ change.
 
 ### Soft-deleted resources
 
-Some APIs need to "[soft delete][aip-135]" resources, marking them as deleted
-or pending deletion (and optionally purging them later).
+Some APIs need to "[soft-delete][]" resources, marking them as deleted or
```

Comment:
as mentioned in the delete AEP: I think we should drop the pattern entirely and replace it.

But if it's too much trouble, I could do that in my soft delete proposal.

Reply:
I think we should leave this guidance in until we fully excise soft-delete...are you okay with removing this when we merge the alternative?

Reply:
yes.